File: Vol.03(1200-1299)/#1293 Floppies/p1293.cpp
// 2905278 2008-05-11 00:25:16 Accepted 1293 C++ 00:00.00 832K 呆滞的慢板 

// 很弱的题，除法，对于不整除者进一处理 

#include <iostream>
using namespace std;

int main() {
    
    int     N,  T = 0;
    
    while( ( cin >> N ) && N ) {
        
        cout << "File #" << ++T << endl;
        
        N = ( N + 1 ) / 2;
        N = N + ( N + 1 ) / 2;
        
        if( N % 62 )    N = N / 62 + 1;
        else            N /= 62;
        
        if( N % 30000 ) N = N / 30000 + 1;
        else            N /= 30000;
        
        cout << "John needs " << N << " floppies.\n" << endl;
        
    }
    
}

File: 范例/Accpt/12XX/p1293.pas
PROGRAM p1293;

VAR
        step                                    :Integer;
        n                               :Longint;

BEGIN
step:=0;
readln(n);
while n<>0 do
        begin
        inc(step);
        {n:=round(n/2);
        n:=round(n*1.5);
        }
        n:=round(n*0.75);
        if frac(n/62/30000)=0 then n:=trunc(n/62/30000)
                else n:=trunc(n/62/30000)+1;
        writeln('File #',step);
        writeln('John needs ',n,' floppies.');
        writeln;
        readln(n);
        end;
END.
